apps/studio/src/lib/menu/tableMenu.ts

```diff
@@ -130,6 +130,53 @@ export async function copyRange(options: {
   options.range.getElement().classList.add("copied");
 }
 
+export function pasteRange(range: Tabulator.RangeComponent) {
+  const text = ElectronPlugin.clipboard.readText();
+  if (!text) return;
+
+  const parsedText = Papa.parse(text, {
+    header: false,
+    delimiter: '\t',
+  });
+
+  if (parsedText.errors.length > 0) {
+    const cell = range.getCells()[0]
+    setCellValue(cell, text);
+  } else {
+    const table = range.getCells()[0].getTable()
+    const rows = table.modules.spreadsheet.getRows().slice(range.getTop())
+    /* eslint-disable-next-line @typescript-eslint/ban-ts-comment */
+    // @ts-ignore
+    // TODO: use range.getLeft(), which is not available in tabulator yet
+    const columns = table.modules.spreadsheet.getColumns().slice(range._range.left + 1)
```

Comment:
Is this part of the PR on the tabulator side?

Reply:
Yeah. It's an easy fix.